sdk/js/src/relayer/relayer/info.ts

```diff
@@ -154,32 +215,195 @@ export async function getWormholeRelayerInfo(
+            const tokenContract = ERC20__factory.connect(
+              "0x" + parsedTokenInfo.originAddress.substring(24),
+              tokenProvider
+            );
+            tokenName = await tokenContract.name();
+            tokenSymbol = await tokenContract.symbol();
+            tokenDecimals = await tokenContract.decimals();
+          } catch (e) {
+            console.log(e);
+          }
+          // Try to get wormscan information on if the tokens have been signed
+          try {
+            const tokenVaa = await getWormscanInfo(
+              environment,
+              sourceChain,
+              parseInt(tokenBridgeLog.sequence),
+              CONTRACTS[environment][sourceChain].token_bridge || ""
+            );
+            signedVaaTimestamp = new Date(
+              (await tokenVaa.json()).data?.indexedAt
+            ).getTime();
+          } catch {}
+
+          const parsed: TokenTransferParsed = {
+            amount: BigNumber.from(parsedTokenInfo.amount)
+              .mul(
+                BigNumber.from(10).pow(
+                  tokenDecimals && tokenDecimals > 8 ? tokenDecimals - 8 : 1
+                )
+              )
+              .toBigInt(),
+            originAddress: parsedTokenInfo.originAddress,
+            originChain: parsedTokenInfo.originChain,
+            targetAddress: parsedTokenInfo.targetAddress,
+            targetChain: parsedTokenInfo.targetChain,
+            fromAddress: parsedTokenInfo.fromAddress,
+            name: tokenName,
+            symbol: tokenSymbol,
+            decimals: tokenDecimals,
+            signedVaaTimestamp,
+          };
+          return parsed;
+        } else if (messageKey.keyType === 2) {
+          // check receipt
+          const cctpKey = parseCCTPKey(messageKey.key);
+          const cctpInfo = await getCCTPMessageLogURL(
+            cctpKey,
+            sourceChain,
+            receipt,
+            environment
+          );
+          const url = cctpInfo?.url || "";
+
+          // Try to get attestation information on if the tokens have been signed
+          let attested = false;
+          try {
+            const attestation = await fetch(url);
+            attested = (await attestation.json()).status === "complete";
+          } catch (e) {
+            console.log(e);
+          }
+          const cctpLog = cctpInfo?.cctpLog!;
+          const parsed: CCTPTransferParsed = {
+            amount: BigNumber.from(
+              Buffer.from(cctpLog.data.substring(2, 2 + 64), "hex")
+            ).toBigInt(),
+            mintRecipient: "0x" + cctpLog.data.substring(2 + 64 + 24, 2 + 128),
+            destinationDomain: BigNumber.from(
+              Buffer.from(cctpLog.data.substring(2 + 128, 2 + 192), "hex")
+            ).toNumber(),
+            attested,
+            estimatedAttestationSeconds: estimatedAttestationTimeInSeconds(
+              sourceChain,
+              environment
+            ),
```

Comment:
got a parser for circle messages: https://github.com/wormhole-foundation/connect-sdk/blob/develop/platforms/evm/src/protocols/circleBridge.ts#L173-L175

Reply:
Ah - I think it might be a bit odd to import the new SDK into the old SDK :P 

In reality, probably what should have been done with AR is for it to have it's own suite of helpers living somewhere, and it imports the new SDK. But, since the current home is the old sdk, that makes things tricky :(